week-4/implement/repeat.test.js

```diff
@@ -23,3 +23,12 @@
 // Given a target string str and a negative integer count,
 // When the repeat function is called with these inputs,
 // Then it should throw an error or return an appropriate error message, as negative counts are not valid.
+function rePeat(str, count) {
+    for (let i = 0; i < count; i++) {
+      if (count < 0) {
+        console.log("Error: Your input is negative"); // Error Message still not working //
+      }
+      else {console.log(str);}
+  }
+  }
+  rePeat("My name is Haythem. I am a software developer in CYF", 3);
```

Comment:
I think the issue  here is  your code (if (count < 0)) is inside the loop. try putting it outside the loop, you might want to use <= instead of < to include the case where count is 0.for example:   function rePeat(str, count) {
    if (count <= 0) {
        console.log("Error: Your input is non-positive");
    } else {
        for (let i = 0; i < count; i++) {
            console.log(str);
        }
    }
}

rePeat("My name is Haythem. I am a software developer in CYF", 3);